### ISSUE TITLE
Projects to be added

### ISSUE THREAD

**Issue by jancborchardt** [libreprojects/libreprojects#8]
Projects which are open source & hosted. Only need to get an icon and add it to the [JSON file](https://github.com/libreprojects/libreprojects/blob/gh-pages/js/data.json)
- [JS Bin](http://jsbin.com/) ([MIT](https://github.com/remy/jsbin/))
- [Hastebin, MIT](http://hastebin.com/)
- [Clipperz](http://clipperz.com) (AGPLv3)
- [Substance](http://substance.io/) (GPLv3)
- [Sharetribe](https://www.sharetribe.com/) ([MIT](https://github.com/sharetribe/sharetribe)
- [I AM CC](http://i-am-cc.org/)
- [Coworfing](http://coworfing.com/) ([GPLv3](https://github.com/nukomeet/coworfing))
- [Tranquillpoet](http://tranquillpoet.com/) ([MIT](https://github.com/bilalq/Tranquillity-Editor))
- [Persona](http://persona.org) (add it when more sites accept it)
- [Mailoo](https://www.mailoo.org/) (uses [only free software](https://www.mailoo.org/legal.php))
- [jslander](http://somethinghitme.com/projects/jslander/) (See @loktar00 [comment](https://github.com/libreprojects/libreprojects/issues/19#issuecomment-9926640))
## Added
- [Terms of Service; Didn’t Read](http://tos-dr.info/) (AGPL)

**Comment by jhuet** [comment]
The question about Persona is a good one but so far we already have [OnlineGroups](http://libreprojects.net/#project=onlinegroups) that runs on [GroupServer](http://groupserver.org) wich already is [not really easy](http://groupserver.org/downloads/install) to get working.

**Comment by jancborchardt** [comment]
Yeah, my concern about the infrastructure-nature of Persona is mostly: Is it really a web service? I guess it would fit in the »Tools« category?

**Comment by jhuet** [comment]
Ok i understand. Persona is an implementation of BrowserID protocol. So, to help us answer that question, would we one day accept something like myopenid.com in here too as it implements OpenID protocol ?

Personnaly, i'm not sure it fits.

**Comment by jancborchardt** [comment]
Yes, I agree. Essentially the goal of it is that your email provider uses BrowserID itself anyway. Maybe we should add it once more sites use BrowserID.

**Comment by airon90** [comment]
http://code.google.com/p/bobelisk/ (GPL)

**Comment by jancborchardt** [comment]
I had that on the list before but decided to not add it because it’s way
too nerdy (although admittedly, I had fun playing it).

**Comment by airon90** [comment]
http://ur1.ca/ (URL shortening, GPL)

**Comment by jancborchardt** [comment]
@Airon90 if you’re feeling awesome, you can also just start adding the projects to the site. Ideally one project per commit, full with info in the [JSON file](https://github.com/libreprojects/libreprojects/blob/gh-pages/js/data.json) and image. Best fork it and issue a pull request first (instead of pushing directly to gh-pages), then @jhuet and I can review it before. :)
Same goes for #19 (Games) and #30 (OpenPhoto & Pixlpaste), would be cool if you want to start diving into the code.

**Comment by airon90** [comment]
Sorry for my silence but university is killing me slowly O_O
@jancborchardt  Sincerely I never used Github :D I fork with the button above, I hack the code, then? :D

**Comment by jancborchardt** [comment]
Once http://www.crowdhoster.com/ is out of the invite-only phase, we could add it. It’s a crowdfunding service based on the open source http://selfstarter.us – and the Crowdhoster folks seem to contribute back to it:

> And to the open source community: please join us in continuing to make the Selfstarter project better and better. We're picturing analytics tools, layout themes, plugins...the sky is the limit.

**Comment by jancborchardt** [comment]
We don’t list https://ethercalc.org yet, the spreadsheet cousin of Etherpad/Titanpad.

**Comment by yuwash** [comment]
ethercalc: That's a nice project! Though I'd wish a way to download my spreadsheet in a format that I can import in local spreadsheet software to use interactively (i. e. with formulas), the project does seem to be already mature enough for productive usage in many cases where csv or html suffices. Moreover, the github project [audreyt/ethercalc](https://github.com/audreyt/ethercalc) is fortunately quite active.

**Comment by yuwash** [comment]
What about [PasteVault](https://www.pastevault.com)? The MIT-licensed project is [UserScape/PasteVault](https://github.com/UserScape/PasteVault)

I assume there are also other similar projects (if you know anything better, I'd be happy to know about it), but libreprojects currently doesn't list any service for encrypted text hosting yet.

**Comment by jancborchardt** [comment]
@yuwash PasteVault uses server side encryption so it’s not completely secure – the server can know the password. Actually even when using client side encryption it wouldn’t be secure, so I think leading people to believe it’s secure is not that cool.

**Comment by yuwash** [comment]
@jancborchardt Ok, I understand client-side encryption is preferable. I've been using http://protectedtext.com , which is, however, not floss. I just thought I found this as a free alternative. I don't have profound knowledge in encryption or network security, but aren't these services at least considerably more secure than e. g. saving your data in your web mail account or a cloud storage? Or, what would be your suggestion for people who don't have their own servers?

**Comment by jancborchardt** [comment]
@yuwash I would recommend [ownCloud](http://owncloud.org) (there are free providers) but of course I’m biased. ;) It’s always a convenience-security tradeoff though, yeah.

**Comment by yuwash** [comment]
What about these simple image/media hosting sites
- **[img.bi](https://img.bi)**, source: https://github.com/imgbi
- **[MediaCrush](https://mediacru.sh)**, source: https://github.com/MediaCrush/MediaCrush

I've found them at [PRISM BREAK](https://www.prism-break.org/)

**Comment by jancborchardt** [comment]
@yuwash yeah, img.bi looks good! Can you add it to the »tools« category?

**Comment by jancborchardt** [comment]
By the way, some new stuff that is in development but about to be fit for adding:
- [Poche](http://inthepoche.com), read-it-later app: http://github.com/inthepoche/poche
- [Codebox](http://codebox.io), cloud IDE: http://github.com/FriendCode/codebox

**Comment by ironbishop** [comment]
btw, we already have Cloud9 that's very (very!) similar to Codebox (and cheaper, IMHO)

**Comment by yuwash** [comment]
On the other hand, Poche, which is a good alternative in a field where I didn't know of any open-source alternative until now.

Generally, choice is not a bad thing, because being open-source isn't everything, the service should be in the style that the user feels right with.

Just a note to the above double commits: the first one mistakingly directed to gh-pages branch was a mistake. The [pull request 76](https://github.com/libreprojects/libreprojects/pull/76) comes from my imgbi branch, where the commit e357a877d9cc55be464722d82264443bec6debf9 belongs to, so it's clean.

**Comment by jancborchardt** [comment]
@ironbishop yeah, I know about Cloud9 vs Codebox. Cloud9 is very focused and great for JS or NodeJS projects. The thing is that Codebox offers VMs which is way more powerful for developers. For example, I can’t develop ownCloud using Cloud9, but with Codebox that’s possible. Also I find the interface much cleaner and simpler. Maybe in this case it even warrants adding both?

@yuwash do you also want to prepare a pull request to add Poche? Probably »tools« would be the right category as well.

**Comment by ironbishop** [comment]
Ok. In fact I have to use OpenShift for running the applications I write on Cloud9.

**Comment by jancborchardt** [comment]
@ironbishop yeah – and for me it doesn’t work at all since they only offer a subset of git. For example I can’t clone a project inside a project or use git submodule. That makes it impossible to develop on ownCloud, unfortunately.

**Comment by jancborchardt** [comment]
http://www.framadate.org/index.php?lang=en_GB is similar to Doodle. But it seems not very well localized, as French always comes up again. Any other Doodle alternatives?

**Comment by yuwash** [comment]
[AlternativeTo.net lists as alternatives to Doodle](http://alternativeto.net/software/doodle/?license=opensource), among others, the following:
- [Dudle](https://dudle.inf.tu-dresden.de/) :bell: French :bell: many others :no_bell: pretty :bell: password- protection possible :bell: no email necessary
- Framadate :bell: French :bell: 3 others :no_bell: email needed :bell: pretty
- [Nuages](https://dudle.inf.tu-dresden.de/) :bell: Frennch :bell: 3 others :bell: pretty :no_bell: account with email needed

**Comment by jancborchardt** [comment]
- [Mondrian](http://mondrian.io), a vector editor similar to SVG-edit but better designed. [MIT-licensed](https://github.com/artursapek/mondrian)
